utilities/_utilities.responsive-spacings.scss

```diff
@@ -9,8 +9,8 @@
  * maps, e.g.:
  *
  *   $inuit-responsive-spacing-directions: (
- *     null: null,
- *     bottom: bottom,
+      null: null,
```

Comment:
The leading `*` is missing (Sorry for the nitpicking).

Reply:
Done 😄

Reply:
Thanks 😄